### DISCUSSION TITLE
Problem with the way we use data and GIS

### DISCUSSION THREAD

**Discussion by KenNL42** [albertkun/21S-ASIAAM-191A#23] [Past Thinking Caps]
I believe that using data and GIS must be done while respecting people privacy. This may not be directly related to users who work with the given data, but we still need to be ethical to exclude any information that may violate privacy. For example, we should not use a data that has people and their address. As a data collector, we must also be ethical to exclude data that violates privacy. This is because private data is easy to be leaked. Raw data should not have sensitive information in order to prevent data leakage. Thus, certain data that contains sensitive data should be used ethically and does not violate people privacy.

**Comment by avaitt** [comment]
Hi!

I completely agree that data often can infringe on user privacy. With the era of "big data" that we live in, it's no surprise that targeted advertisements are so accurate and match our wants/needs so well. Going forward as data scientists I think it is of paramount importance that before we create beautiful graphs and visualizations, we first learn to become ethical data collectors. Although likely a tedious process, I think ethical data collection (and in turn ethical data science) helps us in the long term. Building good data practices and habits ensures we use our knowledge and skillset to not cause harm.

**Comment by nnhimichelle** [comment]
I agree with this fully!! With the world of the web that we live in, it feels incredibly difficult to escape personalized content that is privacy-invasive. Ethics is incredibly important, but it feels barely acknowledge when it comes to data collection, such as one's IP address. More emphasis on ethics will definitely make data collection better in the long-term, but it would also make people feel more comfortable in general.
